public/js/monitor.js

```diff
@@ -75,9 +55,24 @@ function showFalseDoor(){
 
 }
 
+async function sha1(message) {
+  const msgBuffer = new TextEncoder("utf-8").encode(message);
+  const hashBuffer = await crypto.subtle.digest("SHA-1", msgBuffer);
+  const hashArray = Array.from(new Uint8Array(hashBuffer));
+  const hashHex = hashArray.map(b => ("00" + b.toString(16)).slice(-2)).join("");
```

Comment:
Could we use something like [`String#padStart()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) here?

Not sure if that'd reduce it to something like this, or if there is something else I'm not  understanding:
```js
const hashHex = hashArray.map(b => b.toString(16).padStart(2, "0")).join("");
```

Reply:
I couldn't make this work quickly ... keeping the way I have it.